docs/central-users.rst

```diff
@@ -5,92 +5,132 @@ Managing Users in Central
-| **Project App Users**                                                                |
-+------------------+---------------+-----------------+----------------+----------------+
-| Create           | x             | x               |                |                |
-+------------------+---------------+-----------------+----------------+----------------+
-| List All         | x             | x               |                |                |
-+------------------+---------------+-----------------+----------------+----------------+
-| Revoke Access    | x             | x               |                |                |
-+------------------+---------------+-----------------+----------------+----------------+
-| See Code         | x             | x               |                |                |
-+------------------+---------------+-----------------+----------------+----------------+
-| **Web Users**                                                                        |
-+------------------+---------------+-----------------+----------------+----------------+
-| Create           | x             |                 |                |                |
-+------------------+---------------+-----------------+----------------+----------------+
-| List All         | x             |                 |                |                |
-+------------------+---------------+-----------------+----------------+----------------+
-| Edit Details     | x             |                 |                |                |
-+------------------+---------------+-----------------+----------------+----------------+
-| View Email Addr. | x             |                 |                |                |
-+------------------+---------------+-----------------+----------------+----------------+
-| Revoke Password  | x             |                 |                |                |
-+------------------+---------------+-----------------+----------------+----------------+
-| Delete           | x             |                 |                |                |
-+------------------+---------------+-----------------+----------------+----------------+
+Central features Role-based User permissioning for Web Users. Web Users can either be system-level administrators which can perform all actions on all resources across the system, or they can be given specific Roles in one or more Projects.
+
+Central currently defines three Project-level Roles:
+
+- **Project Managers** can perform any action on their assigned Project
+- **Project Viewers** only see created Forms and Submissions within the Project they are assigned to, and cannot edit anything
+- **Data Collectors** may see the list of all Forms in the Project, and retrieve basic information about them, but may only create new Submissions. They cannot see any Submissions. This Role is similar to :ref:`App Users <central-users-app-overview>` except that Data Collectors always see every Form in a Project and only use Web Forms.
```

Comment:
As of .4, they will no longer see closed forms, but that's probably a level of detail that's not needed here.